rendering/cases/rtl-text-align/main.js

```diff
@@ -0,0 +1,192 @@
+import CircleStyle from '../../../src/ol/style/Circle.js';
+import Feature from '../../../src/ol/Feature.js';
+import Fill from '../../../src/ol/style/Fill.js';
+import Map from '../../../src/ol/Map.js';
+import Point from '../../../src/ol/geom/Point.js';
+import Stroke from '../../../src/ol/style/Stroke.js';
+import Style from '../../../src/ol/style/Style.js';
+import Text from '../../../src/ol/style/Text.js';
+import VectorLayer from '../../../src/ol/layer/Vector.js';
+import VectorSource from '../../../src/ol/source/Vector.js';
+import View from '../../../src/ol/View.js';
+
+const vectorSource = new VectorSource();
+let feature;
+
+// Latin - end (right)
+feature = new Feature({
+  geometry: new Point([-10, 50]),
+});
+feature.setStyle(
```

Comment:
Setting a style function on the `VectorSource` and the `text` / `textAlign` on the `Feature`s might make it easier to see what's being tested here.

Reply:
Good suggestion! Implemented with 1db2c4bc45.